components/calendar/src/julian.rs

```diff
@@ -179,6 +132,46 @@ impl Julian {
 
         prev_month_days + date.day.0 as u32
     }
+
+    fn calculate_slack_from_iso(date: IsoDateInner) -> u32 {
```

Comment:
What's this function attempting to do?

Reply:
To calculate what is the difference between georgian and julian calendars.

March 1st 200 should give -> 0
today() should give -> 13

Reply:
Gotcha. I suspect this can simply be the implementation code in the to/from iso conversion functions in the `Calendar` impl?